windowspkg/linkerd2-cli/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,14 @@
+﻿$ErrorActionPreference = 'Stop';
+$toolsPath = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+$packageArgs = @{
+
+  packageName    = 'linkerd'
+  fileFullPath   = "$toolsPath\linkerd.exe"
```

Comment:
Is there a way to set this value from an env var so that user can decide where to download the `.exe` to?

Reply:
Yup, if required the script can also accept command line arguments for the path. Should I accommodate for that as well?

Reply:
Even better! Thanks!